networkx/algorithms/disjoint_paths/simple_disjoint.py

```diff
@@ -0,0 +1,73 @@
+"""Disjoint paths computation using removal of found shortest paths.
+"""
+
+#    Copyright (C) 2015-2016 by
+#    Niels L. M. van Adrichem <n.l.m.vanadrichem@tudelft.nl>
+#    All rights reserved.
+#    BSD license.
+
+__author__ = """\n""".join(['Niels L. M. van Adrichem <n.l.m.vanadrichem@tudelft.nl>'])
+__all__ = ['simple_disjoint']
+
+import networkx as nx
+
+def simple_disjoint(G, source, target, weight=None, k=2, node_disjoint=False):
+    """ Simplest, even naive, disjoint path computation by removing previously
+    found paths and computing the next shortest path. This method, although it 
+    often works out, gives no guaranteed correct result since it is susceptibl
+    e to trap topologies where the removal of the shortest path blocks finding
+    2 non-shortest path disjoint paths. Furthermore, besides the first
+    found path equalling the shortest path, it doesn't optimize for path
+    lengths.
+
```

Comment:
The docstring needs improvement, e.g. list of parameters, examples, etc. The first sentence should summarize the rest and be followed by a more detailed explanation, with a blank line between both. See https://github.com/networkx/networkx/wiki#user-content-docstring-formatting

Also, there should be two blanks between two sentences.